src/Components/AstroChat/AstroChat.tsx

```diff
@@ -91,14 +93,28 @@ export const AstroChat: React.FunctionComponent<AstroChatProps> = ({
   ) : (
     <>
       <Alert
-        className="pf-v5-u-mb-md"
+        className="astro-v5-c-alert-welcome pf-v5-u-background-color-200 pf-v5-u-mb-md"
         variant="info"
         isInline
         title="You are about to utilize Red Hat's Hybrid Cloud Console virtual assistant chat tool"
+        actionClose={<AlertActionCloseButton onClose={() => console.log('Clicked the close button')} />}
       >
         Please do not include any personal information or confidential information in your interaction with the virtual assistant. The tool is
-        intended to assist with general queries. Please see <a href="https://www.redhat.com/en/about/terms-use">Red Hat Terms of use</a> and{' '}
-        <a href="https://www.redhat.com/en/about/privacy-policy">Privacy Statement</a>.
+        intended to assist with general queries.
+        <div className="pf-v5-u-mt-md">
+          <a href="https://www.redhat.com/en/about/terms-use" className="pf-v5-u-pr-md">
+            Red Hat Terms of use{' '}
+            <Icon iconSize="sm" isInline>
+              <ExternalLinkAltIcon />
+            </Icon>
+          </a>
+          <a href="https://www.redhat.com/en/about/privacy-policy">
+            Privacy Statement{' '}
+            <Icon iconSize="sm" isInline>
+              <ExternalLinkAltIcon />
+            </Icon>
+          </a>
+        </div>
       </Alert>
```

Comment:
To make the alert dismissable, you can wrap it in a state variable (something like `alertClosed`) and update that variable (`setAlertClosed`) within `actionClose`. That would look something like this:

```
      {!alertClosed && (
        <Alert
          className="astro-v5-c-alert-welcome pf-v5-u-background-color-200 pf-v5-u-mb-md"
          variant="info"
          isInline
          title="You are about to utilize Red Hat's Hybrid Cloud Console virtual assistant chat tool"
          actionClose={<AlertActionCloseButton onClose={() => setAlertClosed(true)} />}
        >
          Please do not include any personal information or confidential information in your interaction with the virtual assistant. The tool is
          intended to assist with general queries.
          <div className="pf-v5-u-mt-md">
            <a href="https://www.redhat.com/en/about/terms-use" className="pf-v5-u-pr-md">
              Red Hat Terms of use{' '}
              <Icon iconSize="sm" isInline>
                <ExternalLinkAltIcon />
              </Icon>
            </a>
            <a href="https://www.redhat.com/en/about/privacy-policy">
              Privacy Statement{' '}
              <Icon iconSize="sm" isInline>
                <ExternalLinkAltIcon />
              </Icon>
            </a>
          </div>
        </Alert>
      )}
```

Afterwards, you would just need to add that state on line 47:
```
const [alertClosed, setAlertClosed] = useState<boolean>(false);
```